### ISSUE TITLE
could StExBar support run cmd as administrator

### ISSUE THREAD

**Issue by daqingshu** [stefankueng/tools#26]
tks a lot

**Comment by joschmid67** [comment]
Yes, that would be very helpful! Ideally a checkbox for every command for RunAs Administrator.

**Comment by stefankueng** [comment]
hold down the Win key while you click on the tool button - it will run as Admin.

**Comment by daqingshu** [comment]
> hold down the Win key while you click on the tool button - it will run as Admin.

wow, thanks a lot.
